.brigade/project.yaml

```diff
@@ -1,5 +1,5 @@
-# yaml-language-server: $schema=https://raw.githubusercontent.com/brigadecore/brigade/v2/v2/apiserver/schemas/project.json
-apiVersion: brigade.sh/v2-beta
+# yaml-language-server: $schema=https://schemas.brigade.sh/schemas-v2/project.json
```

Comment:
Nice.  Is this URL served via a DNS entry pointing to the raw GH URL?  Or do we upload schemas somewhere and serve?

Reply:
DNS tricks would only get us so far-- they'd let us swap `raw.githubusercontent.com` for `schemas.brigade.sh`, but wouldn't let us improve the path.

I created an Azure storage account for schemas.

Refreshing the contents of that on release isn't automated yet, but that is for sure something we can do in the near future.

Reply:
btw... I wanted `https://schemas.brigade.sh/v2/project.json` but annoyingly, v2 is not a valid bucket name because it's too few characters. 🤷‍♂️